fhirstore/search/formatter.py

```diff
@@ -0,0 +1,64 @@
+from fhirstore.search.url_parser import URL_Parser
+import elasticsearch
+import logging
+
+
+class Formatter:
+    def __init__(self):
+        self.hits = {}
+        self.included_hits = {}
+        self.bundle = {}
+
+    def parse_format_arguments(self, url_args, resource_type):
+        parsed_args = URL_Parser(url_args, resource_type)
```

Comment:
Do you really need to use the Parser inside the Formatter?
I think it's better if you separate all the concerns and call these only in the "main" function

Reply:
I changed that, it is now done in fhirstore.py